theano/gpuarray/dnn.py

```diff
@@ -1427,10 +1427,10 @@ def make_node(self, dy, sm):
         return Apply(self, [dy, sm], [sm.type()])
 
 
-@local_optimizer([AbstractConv2d, AbstractConv2d_gradWeights,
-                  AbstractConv2d_gradInputs])
+@op_lifter([AbstractConv2d, AbstractConv2d_gradWeights,
```

Comment:
Wrapping this in an op_lifter isn't useful as we don't register the op_lifter. It don't cause problems, but isn't useful.